Project.toml

```
@@ -1,7 +1,7 @@
 name = "Meshes"
 uuid = "eacbb407-ea5a-433e-ab97-5258b1ca43fa"
 authors = ["Júlio Hoffimann and contributors"]
-version = "0.40.0"
+version = "0.41.0"
```

```suggestion
version = "0.40.1"
```
We are currently using minor releases for breaking changes (we are still v0.y.z)